### PR TITLE
Transfer `aria-label` from nav settings menu to settings menu icon-button

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Signed-off-by: julia.kirschenheuter <julia.kirschenheuter@nextcloud.com>

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1: as discussed, thanks

**Review by artonge — APPROVED** [review]

File: core/templates/layout.user.php [hunk 1]
```diff
@@ -144,7 +144,7 @@
 				</div>
 				<div id="settings">
 					<div id="expand" tabindex="0" role="button" class="menutoggle"
-						aria-label="<?php p($l->t('Settings'));?>"
+						aria-label="<?php p($l->t('Settings menu'));?>"
```
Comment (artonge) [inline core/templates/layout.user.php:147]: Actually, shouldn't it be something like that:

```
 						aria-label="<?php p($l->t('Open settings menu'));?>"
```

**Review by PVince81 — APPROVED** [review]
:+1:

File: core/templates/layout.user.php (hunk 1 above)
Comment (PVince81) [reply core/templates/layout.user.php]: @artonge it's done, please approve

**Review by artonge — APPROVED** [review]

**Comment by artonge** [comment]
CI failure seems unrelated

**Pull request merged**